dll/win32/shell32/utils.cpp

```diff
@@ -297,3 +297,35 @@ LargeIntegerToString(
     return Int64ToString(pLargeInt->QuadPart, pszOut, cchOut, bUseFormat,
                          pNumberFormat, dwNumberFlags);
 }
+
+/*************************************************************************
+ *                PathIsEqualOrSubFolder (SHELL32.755)
+ */
+EXTERN_C
+BOOL WINAPI PathIsEqualOrSubFolder(_In_ LPCWSTR pszFile1OrCSIDL, _In_ LPCWSTR pszFile2)
+{
+    WCHAR szCommon[MAX_PATH], szFile1[MAX_PATH];
```

Comment:
We should really be moving away from using MAX_PATH everywhere in the shell, not adding new cases every day...

Reply:
I can remove `MAX_PATH` limit, but it increases the program size...

Reply:
Added `<DX_Shell.h>` to remove `MAX_PATH` limit and used it.

Reply:
Hmmm, do we have that header?

Reply:
This header is my invention.